### DISCUSSION TITLE
CP-01 There is no program, support does not understand

### DISCUSSION THREAD

**Discussion by vlkator** [CrealityOfficial/Creality_Sonic_Pad#119] [Q&A]
Hello, I received an older, yet brand new unboxed printer, laser and CNC creality CP-01. I have everything assembled, but according to the instructions, I need a program (slicer) for 3D printing, a "creality workshop" program for CNC and laser engraving. But there is no such program on the creality website and I have not found it anywhere else. 

Why are old programs deleted when the printer is new and functional? Where can I find a program for CNC and laser? Where can the creality workshop be downloaded? 

And also the printer heats the pad, heats the nozzle and freezes. 

Thanks for any advice and help.

**Comment by BriellaBugs** [comment]
I believe it's this google drive folder

https://drive.google.com/drive/folders/1MUCrBgUPlCp6mv2g3-aAlG2do2nq65eA
